frontend/src/App.svelte

```diff
@@ -27,11 +28,13 @@
   let selected_node = null;
 
   let show_menu = false;
-  let show_legend = false;
+  let show_legend = true;
   let show_timebar = false;
 
-  let legend = new Map<string, string>();
-  $: colors = [...legend.entries()];
+  let rainbowTheme = config.Theme.Rainbow;
+  let themeMap = new Map(Object.entries(rainbowTheme));
```

Comment:
Minor detail with the naming:
It is not rainbow anymore, maybe use something more generic since it is customisable also?